src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java

```diff
@@ -17,22 +17,14 @@
 
 import com.github.mvysny.kaributesting.v10.MockVaadin;
 import com.github.mvysny.kaributesting.v10.Routes;
-import com.karankumar.bookproject.backend.entity.Author;
-import com.karankumar.bookproject.backend.entity.Book;
-import com.karankumar.bookproject.backend.entity.Genre;
-import com.karankumar.bookproject.backend.entity.PredefinedShelf;
-import com.karankumar.bookproject.backend.entity.RatingScale;
+import com.karankumar.bookproject.backend.entity.*;
```

Comment:
Could you undo this change, please? We're not using wildcard imports

Reply:
Yes removed all wildcard imports.